notebooks_tsqr/consdb/access_consdb.ipynb

```diff
@@ -34,6 +34,8 @@
     "import sqlalchemy\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    if have_consdb:
</pre><p>If you won't use a method other than ConsDbClient to access the consdb, why not just let the import throw the exception and stop the notebook?  If you are accessing using another method (that works) that you will "have" a consdb even when have_consdb = False?  Generally, I think catching except around an import should only be done for optional features.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I think that's a great point and I will implement that on the next consdb focused PR. That one will have more 'get this specific data and use it for this plot' whereas what I have currently was more of an example from someone else's notebook, where they used that pattern and also had other sources of data.